packages/templates/wasm/assemblyscript/src/workflows/e2e.yaml

```diff
@@ -0,0 +1,22 @@
+name: simple-storage
+jobs:
+  cases:
+    steps:
+    - uri: fs/build
+      method: deployContract
+      input:
+        connection: null
```

Comment:
noob question: do we need to define connection as null here? would it be possible just to ignore it entirely?